src/main/java/org/elasticsearch/indices/recovery/RecoveryState.java

```diff
@@ -400,16 +402,18 @@ public synchronized long time() {
             if (time >= 0) {
                 return time;
             }
-            return Math.max(0, System.currentTimeMillis() - startTime);
+            long t = TimeValue.nsecToMSec(System.nanoTime());
```

Comment:
Why the assignment to `t` here? All of the other replacements didn't use any temporary variables (just curious)